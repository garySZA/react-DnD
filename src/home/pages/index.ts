export * from './Home';
export * from './About';
export * from './Page404';