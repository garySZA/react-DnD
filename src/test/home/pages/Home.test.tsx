import { render } from '@testing-library/react';
import { Home } from '../../../home';

describe('Pruebas en componente <Home />', () => { 
    test('Debe ser igual al snapshot', () => { 
        const { container } = render( <Home /> );

        expect( container ).toMatchSnapshot();
    });
});