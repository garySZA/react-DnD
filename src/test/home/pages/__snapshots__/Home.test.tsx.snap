// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en componente <Home /> Debe ser igual al snapshot 1`] = `
<div>
  <div>
    Home - test
  </div>
</div>
`;
